src/config/layers.js

```diff
@@ -482,6 +483,167 @@ netzkarteShowcases.setChildren([
+        description: 'ch.sbb.construction.unterhalt.bahnhof-desc',
+        legendUrl: '/img/layers/construction/Unterhalt_Bahnhof.png',
+        construction: {
+          art: 'Unterhalt',
+          ort: 'Bahnhof',
+        },
+      },
+    }),
+  ],
+});
+
+export const constrAusbau = new Layer({
+  name: 'ch.sbb.construction.ausbau.group',
+  desc: 'ch.sbb.construction.ausbau.group-desc',
+  visible: true,
+  properties: {
+    hasInfos: true,
+    description: 'ch.sbb.construction.ausbau.group-desc',
+  },
+  children: [
+    new Layer({
+      name: 'ch.sbb.construction.ausbau.uebrige',
+      key: 'ch.sbb.construction.ausbau.uebrige',
+      visible: true,
+      properties: {
+        hasInfos: true,
+        description: 'ch.sbb.construction.ausbau.uebrige-desc',
+        legendUrl: '/img/layers/construction/Ausbau_brigeStandorte.png',
+        construction: {
+          art: 'Ausbau',
+          ort: 'Übrige Standorte',
+        },
+      },
+    }),
+    new Layer({
+      name: 'ch.sbb.construction.ausbau.bahnhof_strecke',
+      key: 'ch.sbb.construction.ausbau.bahnhof_strecke',
+      visible: true,
+      properties: {
+        hasInfos: true,
+        description: 'ch.sbb.construction.ausbau.bahnhof_strecke-desc',
+        legendUrl: '/img/layers/construction/Ausbau_BahnhofundStrecke.png',
+        construction: {
+          art: 'Ausbau',
+          ort: 'Bahnhof und Strecke',
+        },
+      },
+    }),
+    new Layer({
+      name: 'ch.sbb.construction.ausbau.strecke',
+      key: 'ch.sbb.construction.ausbau.strecke',
+      visible: true,
+      properties: {
+        hasInfos: true,
+        description: 'ch.sbb.construction.ausbau.strecke-desc',
+        legendUrl: '/img/layers/construction/Ausbau_Strecke.png',
+        construction: {
+          art: 'Ausbau',
+          ort: 'Strecke',
+        },
+      },
+    }),
+    new Layer({
+      name: 'ch.sbb.construction.ausbau.bahnhof',
+      key: 'ch.sbb.construction.ausbau.bahnhof',
+      visible: true,
+      properties: {
+        hasInfos: true,
+        description: 'ch.sbb.construction.ausbau.bahnhof-desc',
+        legendUrl: '/img/layers/construction/Ausbau_Bahnhof.png',
+        construction: {
+          art: 'Ausbau',
+          ort: 'Bahnhof',
+        },
+      },
+    }),
+  ],
+});
+
+export const constrSingleLayer = new ConstructionLayer({
+  name: 'ch.sbb.construction.single',
+  key: 'ch.sbb.construction.single',
+  visible: true,
+  properties: {
+    hideInLegend: true,
+    popupComponent: 'ConstructionPopup',
+  },
+  children: [constrUnterhalt, constrAusbau],
+  maxResolution: 305.748113141,
+});
+
+export const constrClusterLayer = new ConstructionLayer({
+  name: 'ch.sbb.construction.cluster',
+  key: 'ch.sbb.construction.cluster',
+  visible: true,
+  properties: {
+    hideInLegend: true,
+    popupComponent: 'ConstructionPopup',
+    cluster: true,
+  },
+  children: [constrUnterhalt, constrAusbau],
+  minResolution: 305.748113141,
+});
```

Comment:
So did you try 2 merge those 2 layers in one to load data only once ?

Reply:
not yet

Reply:
since it's not blocker , I'll merge this one , if you have time you can try it  in another PR.
 ok @gevos?

Reply:
> since it's not blocker , I'll merge this one , if you have time you can try it in another PR.
ok @gevos?

This is a good idea and if @mhaertwig is okay with it.

Reply:
Yes, loading the data only once makes sense! @gevos please create a ticket for this.